src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java

```diff
@@ -73,11 +84,21 @@ protected Object doWork() {
         return indexFile.getAbsolutePath();
     }
 
+    private void writeTabixIndex(final TabixIndex idx, final File idxFile) throws IOException {
+        if (!idxFile.getAbsolutePath().endsWith(TabixUtils.STANDARD_INDEX_EXTENSION)) {
+            //Creating tabix indices with a non standard extensions can cause problems so we disable it
+            throw new UserException("File " + featureFile + " requires a tabix index file with a \"" + TabixUtils.STANDARD_INDEX_EXTENSION + "\" extension");
+        }
+        try (final LittleEndianOutputStream stream = new LittleEndianOutputStream(new BufferedOutputStream(new BlockCompressedOutputStream(idxFile)))){
```

Comment:
Did some digging, and it looks like this behavior is correct -- the tbi index itself is supposed to be block compressed. Cf., eg., the code for writing tbi indices in htslib (https://github.com/samtools/htslib/blob/1e2ed48935a58845cecb64880e8cf7553c7db852/hts.c):

    } else if (fmt == HTS_FMT_TBI) {
        BGZF *fp = bgzf_open(fnidx, "w");
        if (fp == NULL) return -1;
        bgzf_write(fp, "TBI\1", 4);
        hts_idx_save_core(idx, fp, HTS_FMT_TBI);
        bgzf_close(fp);